src/docs/asciidoc/index.adoc

```diff
@@ -306,6 +306,17 @@ The https://plugins.gradle.org/plugin/io.micronaut.library[Micronaut library plu
 * Applies the https://search.maven.org/artifact/io.micronaut.platform/micronaut-platform[Micronaut Bill of Materials (BOM)]
 * Applies the `java-library` plugin
 * Configures annotation processing for the current language (Groovy, Java or Kotlin)
+* Adds annotation processors automatically for Micronaut modules
```

Comment:
Can you create a section where the automatic addition of annotation processors is explained and link it from here with the cross reference asciidoc syntax `<<autAnnProcessors,Adds annotation processors automatically for Micronaut modules>>` . 

We need to say something ike when the plugin detects you have a dependency with the data group id , it adds the annotation processor. Check the code to see what it is doing.